.github/workflows/build-documentation.yml

```diff
@@ -167,6 +167,21 @@ jobs:
           # Could add pandoc and convert to github markdown
           # pandoc CFE_Users_Guide.pdf -t gfm
 
+  deploy-usersguide:
```

Comment:
Not sure if this is a merge artifact but there's two "deploy" jobs per guide now. See line 139 and 257. I think what we want is to move the steps from the jobs defined by 139 and 257 and move them into the previous job.

Reply:
Resolved